firmware/cmake/generate_cubemx.mk

```diff
@@ -1,13 +1,12 @@
-ifeq ($(OS),Windows_NT)
-# Convert windows backslash to regular slash
-	CUBEMX_PATH := $(subst \,/,$(shell where STM32CubeMX))
-else # Linux / MacOS
-    CUBEMX_PATH := $(shell which STM32CubeMX)
+space := $(subst ,, )
+CUBEMX_PATH := $(shell which STM32CubeMX)
```

Comment:
sorry, I made a mistake and didn't fully verify that `which` works on windows git bash. The function works and returns `.../STM32CubeMX` but `where` returns `.../STM32CubeMX.exe`. The Java call fails without the .exe extension.

Can you return this section as it was before? Don't combine it with the CUBEMX_JAVA if statement, I think it will be more clear with them handled separately